packages/MorphicAPIExplorer-Tests.package/MEXLinter.class/instance/dontTestAddDependantMethodsLessOrEqualThanRemoveDependents.st

```diff
@@ -0,0 +1,5 @@
+test-for-possible-bugs
+dontTestAddDependantMethodsLessOrEqualThanRemoveDependents
+	"We are adding more dependents in MEXOverview for UI reasons so this test always fails"
```

Comment:
I think this test is about also removing dependents afterwards, which seems to happen nowhere and thus could lead to memory leak ('dangling dependents'). I'm not sure, but simply destroying the super object might not be sufficient. Please clarify, or maybe we should ask the SWT teaching team and open an issue.